mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -233,7 +233,10 @@ public boolean onNavigationItemSelected(MenuItem item) {
     public void setupUserStatusToggle() {
         userStatusToggle
                 = (SwitchCompat) mNavigationHeader.findViewById(R.id.user_status_toggle);
+        final TextView user_status = ButterKnife.
```

Comment:
This method is deprecated in the newer version and Android's `findViewById` can do the same thing https://stackoverflow.com/questions/45516753/butterknife-findbyid-method-is-deprecated-in-version-8-8-0

Also, please follow camelCase for variable-naming

Reply:
sure, will resolve this